bucket/authy.json

```diff
@@ -0,0 +1,63 @@
+{
+    "version": "1.7.2",
+    "description": "Free desktop app for two factor authentication.",
+    "homepage": "https://authy.com/",
+    "license": "Freeware",
+    "architecture": {
+        "64bit": {
+            "url": "https://s3.amazonaws.com/authy-electron-repository-production/authy/stable/1.7.2/win32/x64/authy-electron-1.7.2-full.nupkg",
+            "hash": "sha1:53646438bd9c05efa4790b655ab1d056302a6a7f"
+        },
+        "32bit": {
+            "url": "https://s3.amazonaws.com/authy-electron-repository-production/authy/stable/1.7.2/win32/x32/authy-electron-1.7.2-full.nupkg",
+            "hash": "sha1:adb40cd6a325585b3b2fc32c4ccf29d8d1bb08da"
+        }
+    },
+    "extract_dir": "lib\\net45",
+    "pre_install": [
```

Comment:
Why? 

Any of manifest is doing this. There is no reason to start with this.
There is no research about how applications will handle link. What they will do to this folder and how they accesss them. 

Remove it. Same for uninstaller.